lib/daysOfTheWeek.js

```diff
@@ -9,3 +9,11 @@ module.exports.getDayName = (date) => {
 
   return ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'][passedDate.getDay()];
 };
+
+module.exports.next = (dow) => {
+  const dows = ['Sunday','Monday','Tuesday',  'Wednesday','Thursday', "Friday",'Saturday'];
+  let days = (dows.indexOf(dow)-(new Date()).getDay());
+  if (days < 0) days+=7;
+  if (dows.includes(dow)) return new Date((new Date()).getFullYear(), (new Date()).getMonth(), (new Date()).getDate() + days);
+  throw new Error('no such day');
```

Comment:
This error message is fine, but it looks very different from the error messages in the `getDayName` function.  These should be consistent, so one of the two should be updated.